mindsdb/api/http/namespaces/config.py

```diff
@@ -179,7 +182,18 @@ def get(self):
 
         cloud = ca.config_obj.get('cloud', False)
 
-        return {'mongo': mongo, 'telemtry': telemtry, 'cloud': cloud}
+        local_time = datetime.datetime.now(tzlocal())
+        local_timezone = local_time.tzname()
+
+        return {
+            'mongo': mongo,
+            'telemtry': telemtry,
+            'cloud': cloud,
+            'timezone': {
+                'name': local_timezone,
+                'time': local_time,
```

Comment:
Sending the local time isn't required, timzeone is enough, extra data makes things confusing, just send `'timezone': local_timezone`, and that should suffice :)